docs_app/content/marketing/contributors.json

```
@@ -5,7 +5,7 @@
     "github": "https://github.com/benlesh",
     "picture": "https://avatars2.githubusercontent.com/u/1540597",
     "twitter": "https://twitter.com/BenLesh",
-    "website": "http://www.benlesh.com",
+    "website": "http://benlesh.com",
```

Can we upgrade to https as well?

```suggestion
    "website": "https://benlesh.com",
```